src/thunderbots/software/ai/navigator/path_planner/astar.h

```diff
@@ -0,0 +1,155 @@
+#pragma once
+
+// I apologize if your CLion slows to a crawl
+#include <boost/graph/astar_search.hpp>
+#include <boost/graph/grid_graph.hpp>
+
+#include "ai/world/field.h"
+#include "path_planner.h"
+namespace AStar
+{
+    // TODO: potentially move a lot of stuff out of this header
```

Comment:
How did you want to break stuff up here?

Reply:
I managed to move everything except `AStarGridGraph` into the cpp file